### ISSUE TITLE
DOMDocument::loadXML throws ValueError for empty string since PHP 8

### ISSUE THREAD

**Issue by TimWolla** [php/doc-en#2245]
… and this is not yet documented, neither as a fact in the errors section, nor as a changelog entry.

see:

https://github.com/php/doc-en/blob/7c9f1adb4b7d55a7a37b0503ec895412a6ecc656/reference/dom/domdocument/loadxml.xml#L55-L59

and

https://3v4l.org/qEWjm

**Comment by cmb69** [comment]
Yeah, most `ValueError`s are not documented yet. #163 is still work in progress,
